plugins/pixart-rf/fu-pxi-firmware.c

```diff
@@ -234,3 +248,15 @@ fu_pxi_firmware_new(void)
 {
 	return FU_FIRMWARE(g_object_new(FU_TYPE_PXI_FIRMWARE, NULL));
 }
+
+void
+fu_pxi_firmware_set_is_hpac_flag(FuPxiFirmware *self, gboolean flag)
+{
+	self->is_hpac_flag = flag;
+}
+
+gboolean
+fu_pxi_firmware_get_is_hpac_flag(FuPxiFirmware *self)
+{
+	return self->is_hpac_flag;
+}
```

Comment:
missing newline character at end of file

Reply:
solved in a3f8969ce1bb3ddf34e463aa232dc6861f6f6d3e